lib/handlers/PatchHandler.ts

```diff
@@ -61,8 +62,9 @@ class PatchHandler extends BaseHandler {
       file.upload_length = upload_length
     }
 
+    // TODO: req should be a stream?
     const new_offset = await this.store.write(req, file_id, offset)
```

Comment:
@Acconut @mitjap in the category "how has this ever worked", we have a type error here as we pass a `http.IncomingMessage` to `write` which excepts a `stream.Readable`. All our tests pass streams, not requests, so this is confusing me